tests/e2e/logging/test_openshift-logging.py

```diff
@@ -29,6 +29,13 @@ def setup_fixture(install_logging):
     logger.info("Testcases execution post deployment of openshift-logging")
 
 
+@pytest.mark.skipif(
+    get_ocp_version() == "4.6",
```

Comment:
you can use @skipif_ocs_version('<4.6')

Reply:
@petr-balogh @vavuthu I am checking for OCP version and not OCS.